ESLint/lombiq-base.js

```diff
@@ -0,0 +1,160 @@
+module.exports = {
+    "extends": "./eslint-base.js",
+
+    // Here we can define our own rules overriding the base rules
+    "rules": {
+        // Lombiq rules
+        "max-len": [
+            "error",
+            120,
+            2,
+            {
+                "ignoreUrls": true,
+                "ignoreComments": false,
+                "ignoreRegExpLiterals": true,
+                "ignoreStrings": true,
+                "ignoreTemplateLiterals": true
+            }
+        ],
+
+        "brace-style": [
+            "error",
+            "stroustrup",
+            { "allowSingleLine": true }
+        ],
+
+        "prefer-template": "off",
+
+        "no-plusplus": [
+            "error",
+            { "allowForLoopAfterthoughts": true }
+        ],
+
+        "linebreak-style": ["error", "windows"],
+
+        "no-param-reassign": [
+            "error",
+            {
+                "props": false
+            }
+        ],
+
+        "wrap-iife": [
+            "error",
+            "any", // outside originally
+            { "functionPrototypeMethods": false }
+        ],
+
+        "prefer-arrow-callback": [
+            "error",
+            {
+                "allowNamedFunctions": true, // false originally
+                "allowUnboundThis": true
+            }
+        ],
+
+        "no-underscore-dangle": [
+            "error",
+            {
+                "allowAfterThis": true
+            }
+        ],
+
+        "no-restricted-syntax": [
+            "error",
+            {
+                "selector": "ForOfStatement",
+                "message": "iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations."
```

Comment:
"this guide"?

Reply:
Style guide gondolom, forrás: [airbnb](https://github.com/airbnb/javascript/blob/63098cbb6c05376dbefc9a91351f5727540c1ce1/packages/eslint-config-airbnb-base/rules/style.js#L347)

Reply:
Linkeld be akkor innen.